### ISSUE TITLE
GitHub Actions Fail with duty to create docs on library created with copier_pdm

### ISSUE THREAD

**Issue by WithPrecedent** [pawamoy/duty#13]
**Describe the bug**
Using duty, based on a library created with copier_pdm, I keep running into the same error in GitHub Actions where it can't find the library I've created (after having passed multiple other actions). I'm new to using GitHub Actions and duty - so, I apologize if I'm missing something obvious. But I've spent hours trying to find the problem with no luck. The error looks like this:

```
Run pdm run duty check-docs
  pdm run duty check-docs
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    LANG: en_US.utf-8
    LC_ALL: en_US.utf-8
    PYTHONIOENCODING: UTF-8
    pythonLocation: /opt/hostedtoolcache/Python/3.9.17/x64
    PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.9.17/x64/lib/pkgconfig
    Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.9.17/x64
    Python[2](https://github.com/WithPrecedent/wonka/actions/runs/5466261315/jobs/9950918508#step:6:2)_ROOT_DIR: /opt/hostedtoolcache/Python/[3](https://github.com/WithPrecedent/wonka/actions/runs/5466261315/jobs/9950918508#step:6:3).9.17/x6[4](https://github.com/WithPrecedent/wonka/actions/runs/5466261315/jobs/9950918508#step:6:4)
    Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.9.17/x[6](https://github.com/WithPrecedent/wonka/actions/runs/5466261315/jobs/9950918508#step:6:6)4
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.9.1[7](https://github.com/WithPrecedent/wonka/actions/runs/5466261315/jobs/9950918508#step:6:7)/x64/lib
    LD_PRELOAD: /lib/x[8](https://github.com/WithPrecedent/wonka/actions/runs/5466261315/jobs/9950918508#step:6:8)6_64-linux-gnu/libgcc_s.so.1
> Building documentation
✗ Building documentation (1)
  > mkdocs.build(strict=True, config_file='mkdocs.yml')
  INFO     -  Cleaning site directory
  INFO     -  Building documentation to directory: /home/runner/work/wonka/wonka/site
  ERROR    -  mkdocstrings: No module named 'wonka'
  ERROR    -  Error reading page 'reference/wonka/index.md':
  ERROR    -  Could not collect 'wonka'`
```
**To Reproduce**
I apologize that the steps to reproduce aren't simple because I'm not sure where the error emerges from. The issue arises whenever I push a new version on either the "main" or "development" package of [wonka](https://github.com/WithPrecedent/wonka). Whenever I build the docs locally (which seems to be the step at which the GitHub Action with the corresponding duty fails, I have no problems. Indeed, you can see the manually created docs here: https://withprecedent.github.io/wonka/.

I had made changes to various configuration files, but have reverted them to as close to the created "copier_pdm" template as possible (such as having no "__main__.py" file because this is a library and not a CLI package).
I'd be happy to run any tests of create something smaller to isolate the reproducible error. I'm just not sure where to start.

**Expected behavior**

GitHub Actions complete document creation, and otherwise passes all steps upon a push.

**Screenshots**
I've cut and pasted the error code above. But I can take screenshots as well.

**System (please complete the following information):**
- duty 1.0.0
- python 3.11.3
- linux manjaro kernel 6.1.31-2

**Additional context**
I really appreciate your efforts in creating these tools. They are exactly the sort of things I would create in this context. Thanks!

One thing that I noticed that might be odd is that the error coding refers to "/home/runner/work/wonka/wonka/site." I'm not sure how that directory is created (and if it is just temporary or somewhere on GitPages), but I use a "src/wonka" directory locally and "site" (referring to the created docs site) is built outside of "src."

**Comment by WithPrecedent** [comment]
Of course, after posting the issue, I was finally able to find the solution. 

For anyone else who stumbles across this error and isn't quite sure what to do, this was what I did.

In the pyproject.toml file, I received an error when I had this in the file:
```
[tool.pdm.build]
package-dir = "src/wonka"
```
But, when I removed "wonka" from the dir name, it worked fine. So, it should be:

```
[tool.pdm.build]
package-dir = "src"
```
I'm not sure if my adding of the package's specific name was a holdover from using poetry or why I otherwise added it. Regardless, that is how I fixed the problem.

**Comment by pawamoy** [comment]
Hey @WithPrecedent, thanks for the report and sorry for the trouble. The package-dir must not include the package name indeed, unless it's an implicit namespace package. Glad you found the solution to your issue :tada: Closing :slightly_smiling_face:

**Comment by pawamoy** [comment]
Also, thank you very much for the kind words, and for your demonstration of patience and kindness! That is really appreciated :slightly_smiling_face: Nice project by the way!
